jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/signin.js

```diff
@@ -40,21 +40,14 @@ $(function () {
         var footerContent = '<input  type="submit" id="signin-submit-button" value="' + $labelSignin + '" class="btn btn-primary" name="commit"/>';
 
         var submitDialog = function (e) {
-            if (e.keyCode == enterCode) {
-                //if focus on username then select password field
-                if ($(e.target).is('#userName')) {
+            if(e.which == enterCode) {
+                //simulate submit when enter pressed on "input" element in the internet explorer
+                if($.browser.msie && $(e.target).is(':input')) {
                     e.preventDefault();
```

Comment:
I assume this comment from the commit message should be placed here:
>in internet explorer (specifically ie8) when you press "Enter" button on the item "input", event of "submit" doesn't work constantly, and this element loses focus. So in the "submitDialog" i call the appropriate method manually.

But I don't understand what "doesn't work constantly" means, could you please rephrase a bit and place an inline comment here.